packages/api-explorer/src/components/Select/index.js

```diff
@@ -1,12 +1,27 @@
 import React from 'react'
+import { Select as SelectComponent } from 'antd'
 
-export default function Select({options, onChange}){
+export default class Select extends React.Component {
+
+  shouldComponentUpdate(nextProps) {
```

Comment:
Why we handle directly component update with shouldComponentUpdate? It could be enough extend PureComponent?